ipatests/test_ipalib/test_util.py

```diff
@@ -0,0 +1,44 @@
+# Authors:
+#   Oleg Kozlov <okozlov@redhat.com>
+#
+# Copyright (C) 2018  Red Hat
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test the `ipalib.util` module.
+"""
+
+import os
+import pytest
+from unittest import mock
+
+from ipalib.util import get_pager
+
+
+@pytest.mark.parametrize('pager,expected_result', [
+    # Valid values
+    ('cat', '/usr/bin/cat'),
+    ('/usr/bin/cat', '/usr/bin/cat'),
+    # Invalid values (wrong command, package is not installed, etc)
+    ('cat_', None),
+    ('', None)
+])
+def test_get_pager(pager, expected_result):
+    """
+    Test the `ipalib.util.get_pager` function.
```

Comment:
In general I like the fact that you are adding doc strings. But this doc string just states the obvious and is more of a distraction than a useful explanation. The module name ``ipatests.test_ipalib/test_util`` and function name ``test_get_pager`` already tell a story.